src/Containers.jsx

```diff
@@ -485,6 +488,10 @@ class Containers extends React.Component {
                                                 <div className='pod-name'>{caption}</div>
                                                 <div>{_("pod group")}</div>
                                             </h4>
+                                            <div className='panel-actions'>
```

Comment:
Badge is cool, of course, just maybe it should be the same for container statu as well then?

Reply:
Indeed it looks nicer to be consistent. Fixed.